decidim-assemblies/app/controllers/decidim/assemblies/admin/assemblies_controller.rb

```diff
@@ -17,13 +17,13 @@ def index
         end
 
         def new
-          enforce_permission_to :create, :assembly
+          enforce_permission_to :create, :assembly, assembly: collection.find_by(id: params[:parent_id])
           @form = form(AssemblyForm).instance
           @form.parent_id = params[:parent_id]
         end
 
         def create
-          enforce_permission_to :create, :assembly
+          enforce_permission_to :create, :assembly, assembly: collection.find_by(id: params[:assembly][:parent_id])
```

Comment:
try to make this more DRY, like put in another method for ex. `requested_parent_assembly` or `parent_assembly_from_params`. (not really inspired)

Reply:
New commit to solve this item :)
https://github.com/octree-gva/decidim/commit/93e47eb773e67bde7e3414de35f96f6f87900da5

Reply:
Thanks, looks far better.